client/src/Register.js

```diff
@@ -10,9 +10,12 @@ import ValidationError from './ValidationError';
 function Register() {
   const history = useHistory();
 
+  const [userType, setUserType] = useState(null);
```

Comment:
Instead of having the initial value of userType be null, I'd set it to what you want the default selection to be- either BusinessOwner or Shopper. What will be more common?